utils/email.ts

```diff
@@ -4,3 +4,5 @@ const emailRegex =
 	/^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
 
 export const isValidEmail = (str: string) => emailRegex.test(str);
+export const isValidEmailList = (emailList: string[]) =>
```

Comment:
In my opinion, it would be better to put this function in `utils/validate`, since its usefulness is pretty closely tied to this specific task of form validation (whereas I could see `isValidEmail` being called in all sorts of places).